docs/lib/debugging/fragments/android/dev-menu/setup.md

```diff
@@ -0,0 +1,12 @@
+Run the following commands to add a copy of the file `amplify/.config/local-env-info.json` to the 
+directory `app/src/main/res/raw` in your Android project and rename the copy to `localenvinfo.json`:
+
+```bash
+cd [path to root directory of your Android project]
```

Comment:
Consider just adding a line on top saying "run this from the Android project directory". This way customers can just "copy and paste" and go. Otherwise they'll need to manually use their cursor to move backwards in the command from within the terminal to change the path location.